src/Components/v2/AuctionCard.tsx

```diff
@@ -93,11 +93,13 @@ export const LargeAuctionCard = props => (
     <Serif size="3t" weight="semibold">
       <Truncator maxLineCount={1}>{props.headline}</Truncator>
     </Serif>
-    {!props.isGalleryAuction && !props.isBenefit && (
-      <Serif size="3t">
-        <Truncator maxLineCount={1}>{props.subHeadline}</Truncator>
-      </Serif>
-    )}
+    <Serif size="3t">
+      <Truncator maxLineCount={1}>
+        {!props.isGalleryAuction && !props.isBenefit
+          ? props.subHeadline
+          : "\u00A0"}
```

Comment:
#nonblocking You could change `"\u00A0"` to `<>&nbsp;</>` to make it more readable.